packages/components/typography/stories/typography.stories.js

```diff
@@ -0,0 +1,75 @@
+import { storiesOf, html } from '@open-wc/demoing-storybook';
+import '@tradeshift/elements';
+import '../lib/typography.cjs';
```

Comment:
Let's fix it next week, We can work together.

Reply:
👍 
I don't know what did happen with other components stories actually, and we imported from node_modules